books/free-programming-books-subjects.md

```diff
@@ -136,6 +136,7 @@ Books that cover a specific programming language can be found in the [BY PROGRAM
 
 ### Artificial Intelligence
 
+* [AI Safety for Fleshy Humans](https:/aisafety.dance) - Nicky Case and Hack Club *(:construction: in process)* (CC BY-NC)
```

Comment:
link doesn't work for me.

Reply:
Oh 🫠 I accidentally deleted one "/"/.
Fixing.
Sorry for the trouble!